specs/wp-signup-spec.js

```diff
@@ -168,14 +188,66 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
 										} );
 									} );
 
-									test.describe( 'Step Eight: Can activate my account from an email', function() {
-										test.before( function() {
-											return this.emailClient = new EmailClient( signupInboxId );
+									test.describe( `Step ${stepNum}: Can not publish until email is confirmed`, function() {
+										stepNum++;
+
+										test.it( 'Can see a disabled publish button', function() {
+											const blogPostTitle = dataHelper.randomPhrase();
+											const blogPostQuote = dataHelper.randomPhrase();
+											driver.get( calypsoBaseURL + '/post/' + newBlogAddress );
```

Comment:
This _may_ be better in a page somewhere itself in case we want to reuse it, but we don't currenlty have a create post page (just an editor page which is also used for publishing WordPress pages) so I'm not sure where this would go 🤔